.ci/azure-pipelines/azure-pipelines.yaml

```diff
@@ -1,3 +1,19 @@
+trigger:
+  paths:
+    exclude:
+    - doc
+    - README.md
+    - CHANGES.md
+    - CONTRIBUTING.md
```

Comment:
Does azure-pipelines support wildcard, like `*.md`?

Reply:
I referred this doc and wild cards are not supported with path filters. 
https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/azure-repos-git?view=azure-devops&tabs=yaml#paths